Plugins/Public/base_plugin/Main.cpp

```diff
@@ -2530,6 +2530,93 @@ bool ExecuteCommandString_Callback(CCmds* cmd, const wstring &args)
 		//cmd->Print(L"OK Base is gone are you proud of yourself.");
 		return true;
 	}
+	else if (args.find(L"basedespawn") == 0)
+	{
+		returncode = SKIPPLUGINS_NOFUNCTIONCALL;
+
+		RIGHT_CHECK(RIGHT_BASES)
+
+			uint client = HkGetClientIdFromCharname(cmd->GetAdminName());
+
+		int billythecat = 0;
+		PlayerBase* base;
+		for (auto& i : player_bases)
+		{
+			if (i.second->basename == cmd->ArgStrToEnd(1))
+			{
+				base = i.second;
+				billythecat = 1;
+			}
+		}
+
+
+		if (billythecat == 0)
```

Comment:
can we not add new 'billythecat' code in 2023?
```suggestion
		PlayerBase* base;
		for (auto& i : player_bases)
		{
			if (i.second->basename == cmd->ArgStrToEnd(1))
			{
				base = i.second;
			}
		}


		if (base != nullptr)
```